src/oc_erchef/rebar.config

```diff
@@ -9,7 +9,7 @@
     {lager, ".*",
         {git, "git://github.com/basho/lager.git",                     {branch, "master"}}},
     {bcrypt, ".*",
-        {git, "git://github.com/smarkets/erlang-bcrypt.git",          {branch, "master"}}},
+        {git, "git://github.com/benjamin-bergia/erlang-bcrypt.git",   {branch, "master"}}},
```

Comment:
Should use the original https://github.com/abhay/erlang-bcrypt instead of another fork.

Reply:
Done.